README.md

```diff
@@ -91,6 +91,11 @@ jobs:
     for generating the Connect Gateway endpoint. This only applies if
     "use_connect_gateway" is true. Defaults to auto discovery if empty.
 
+-   `quota_project_id` - (Optional) Project ID from which to pull quota. The
```

Comment:
Should we call this `billing_project_id` to align with gcloud?

https://cloud.google.com/sdk/gcloud/reference#--billing-project

Reply:
There's no consistent name for it:

- billing project
- quota project
- user project
- user quota project

In this case, the API calls to GKE aren't billed (like KMS), so it's not really a "billing" project; it's really just about where quota is allocated.